msgid ""
msgstr ""
"Project-Id-Version: red-discordbot\n"
"POT-Creation-Date: 2021-05-25 15:06+0000\n"
"Last-Translator: \n"
"Language-Team: Turkish\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.3\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: red-discordbot\n"
"X-Crowdin-Project-ID: 289505\n"
"X-Crowdin-Language: tr\n"
"X-Crowdin-File-ID: 670\n"
"Language: tr_TR\n"

#: redbot/cogs/audio/core/events/cog.py:215
msgid "Auto Play started."
msgstr "Otomatik çalma başladı."

#: redbot/cogs/audio/core/events/dpy.py:40
msgid "You have attempted to run Audio's Lavalink server on an unsupported architecture. Only settings related commands will be available."
msgstr "Ses modülünün Lavalink sunucusunu desteklenmeyen bir mimaride çalıştırmayı denediniz. Yalnızca ayarlarla ilgili komutlar kullanılabilir."

#: redbot/cogs/audio/core/events/dpy.py:70
msgid "I'm missing permissions in this server, Please address this as soon as possible.\n\n"
"Expected Permissions:\n"
msgstr "Bu sunucuda gerekli izinlere sahip değilim, lütfen bu sorunu en kısa zamanda çözün.\n\n"
"Beklediğim İzinler:\n"

#: redbot/cogs/audio/core/events/dpy.py:77
msgid "Enabled"
msgstr "Etkinleştirildi"

#: redbot/cogs/audio/core/events/dpy.py:77
msgid "Disabled"
msgstr "Devre dışı bırakıldı"

#: redbot/cogs/audio/core/events/dpy.py:124
msgid "No DJ role found. Disabling DJ mode."
msgstr "DJ rolü bulunamadı. DJ modu kapatılıyor."

#: redbot/cogs/audio/core/events/dpy.py:134
msgid "`{user_input}` is not a valid value for `{command}`"
msgstr "`{user_input}` `{command}` için geçerli bir değer değil"

#: redbot/cogs/audio/core/events/dpy.py:142
msgid "Unable To Parse Argument"
msgstr "Değişken işlenemedi"

#: redbot/cogs/audio/core/events/dpy.py:154
#: redbot/cogs/audio/core/events/dpy.py:163
msgid "Invalid Argument"
msgstr "Geçersiz argüment"

#: redbot/cogs/audio/core/events/dpy.py:155
msgid "The argument you gave for `{}` is not valid: I was expecting a `{}`."
msgstr "`{}` için sunduğunuz değişken geçersiz, ben bir `{}` bekliyordum."

#: redbot/cogs/audio/core/events/dpy.py:175
msgid "Invalid Environment"
msgstr "Geçersiz Ortam"

#: redbot/cogs/audio/core/events/dpy.py:176
msgid "Connection to Lavalink has been lost."
msgstr "Lavalink'e bağlanırken bir bağlantı hatası oluştu."

#: redbot/cogs/audio/core/events/dpy.py:184
msgid "No Player Available"
msgstr "Kullanılabilir müzik çalar yok"

#: redbot/cogs/audio/core/events/dpy.py:185
msgid "The bot is not connected to a voice channel."
msgstr "Bot her hangi bir ses kanalında değil."

#: redbot/cogs/audio/core/events/dpy.py:193
#: redbot/cogs/audio/core/events/lavalink.py:162
msgid "Unable to Get Track"
msgstr "Parça alınamıyor"

#: redbot/cogs/audio/core/events/dpy.py:194
#: redbot/cogs/audio/core/events/lavalink.py:163
msgid "I'm unable to get a track from Lavalink at the moment, try again in a few minutes."
msgstr "Şu anda Lavalink'ten şarkıyı çalamıyorum, lütfen birkaç dakika sonra tekrar deneyin."

#: redbot/cogs/audio/core/events/dpy.py:205
msgid "There was an issue communicating with Discord."
msgstr "Discord ile haberleşirken sorun oluştu."

#: redbot/cogs/audio/core/events/dpy.py:206
msgid "This error has been reported to the bot owner."
msgstr "Hata bot sahibine bildirildi."

#: redbot/cogs/audio/core/events/lavalink.py:154
msgid "Couldn't get a valid track."
msgstr "Geçerli bir parça alınamadı."

#: redbot/cogs/audio/core/events/lavalink.py:189
msgid "Now Playing"
msgstr "Şimdi oynatılıyor"

#: redbot/cogs/audio/core/events/lavalink.py:191
msgid "Track length: {length} | Requested by: {user}"
msgstr "Şarkı uzunluğu: {length} | Şarkıyı sıraya koyan: {user}"

#: redbot/cogs/audio/core/events/lavalink.py:212
msgid "Queue ended."
msgstr "Sıra sonlandırıldı."

#: redbot/cogs/audio/core/events/lavalink.py:264
msgid "Multiple Errors Detected"
msgstr "Birden fazla hata tespit edildi"

#: redbot/cogs/audio/core/events/lavalink.py:265
msgid "Closing the audio player due to multiple errors being detected. If this persists, please inform the bot owner as the Audio cog may be temporally unavailable."
msgstr "Birden çok hata nedeniyle müzik çalar kapatılıyor. Hata devam ederse, lütfen bot sahibini bilgilendiriniz. Audio eklentisi geçici olarak devre dışı kalmış olabilir."

#: redbot/cogs/audio/core/events/lavalink.py:279
msgid "Track Stuck"
msgstr "Parça Takıldı"

#: redbot/cogs/audio/core/events/lavalink.py:280
msgid "Playback of the song has stopped due to an unexpected error.\n"
"{error}"
msgstr "Beklenmedik bir hata nedeniyle şarkıyı oynatmaya son verildi.\n"
"{error}"

#: redbot/cogs/audio/core/events/lavalink.py:286
msgid "Track Error"
msgstr "Şarkı hatası"

